### PR TITLE
Swap camera on mobile devices

### PR DESCRIPTION
**Pull request by ylagef**

### CODE REVIEW

File: src/RTCConfigure.tsx [hunk 1]
```diff
@@ -311,9 +312,30 @@ const RtcConfigure: React.FC<PropsWithChildren<Partial<RtcPropsInterface>>> = (
   // publish local stream
   useEffect(() => {
     async function publish() {
+      const currentPublishedTrack = await client.localTracks.find(
+        (lt) => lt.trackMediaType === 'video'
```
Comment (elenitaex5) [inline src/RTCConfigure.tsx:316]: this parenthesis are really needed?

File: src/TracksConfigure.tsx [hunk 2]
```diff
@@ -1,56 +1,96 @@
...
+
 /**
  * React component that create local camera and microphone tracks and assigns them to the child components
  */
-const TracksConfigure: React.FC<PropsWithChildren<Partial<RtcPropsInterface>>> = (props) => {
+const TracksConfigure: React.FC<
+  PropsWithChildren<Partial<RtcPropsInterface>>
+> = (props) => {
   const [ready, setReady] = useState<boolean>(false)
+  // const mediaStore = useRef<mediaStore>({})
```
Comment (elenitaex5) [inline src/TracksConfigure.tsx]: this comment has to be here? or ccan be removed?

File: src/TracksConfigure.tsx [hunk 3]
```diff
@@ -1,56 +1,96 @@
...
       if (tracks) {
         // eslint-disable-next-line no-unused-expressions
         tracks[0]?.close()
         // eslint-disable-next-line no-unused-expressions
         tracks[1]?.close()
+
+        environmentTrack?.close()
       }
     }
   }, [trackReady, error]) //, ready])
```
Comment (elenitaex5) [inline src/TracksConfigure.tsx]: ready could be removed?

**Review by elenitaex5 — COMMENTED** [review]
check my comments.

File: src/RTCConfigure.tsx (hunk 1 above)
Comment (ylagef) [reply src/RTCConfigure.tsx:316]: Linter adds it

**Review by elenitaex5 — APPROVED** [review]
👍

**Pull request merged**